bin/update_clinvar_schema.py

```diff
@@ -56,9 +57,21 @@ def __init__(self, argv):
 
 		parser.add_argument('-i', dest='input_file', required=True, help='Clinvar XML input file')
 
+		parser.add_argument('-j', dest='java_sources_directory', required=False, \
```

Comment:
"source folder" (instead of "sources directory") is the most common term in English.